examples/svm/plot_custom_kernel.py

```diff
@@ -17,26 +17,26 @@
 iris = datasets.load_iris()
 X = iris.data[:, :2]  # we only take the first two features. We could
                       # avoid this ugly slicing by using a two-dim dataset
-Y = iris.target
+y = iris.target
 
 
-def my_kernel(X, Y):
+def my_kernel(X, y):
```

Comment:
I have made similar changes to the following:
- examples/svm/plot_svm_nonlinear.py
- examples/svm/plot_svm_margin.py
- examples/svm/plot_svm_kernels.py

Do you want me to revert these as well?
I only did this for the sake of consistency with the additions I made.